epub34/a11y/index.html

```diff
@@ -1613,13 +1600,12 @@ <h2>Optimized publications</h2>
 				<li>The EPUB publication should identify the standard or guidelines it follows in a
 						<code>conformsTo</code> property in accordance with [[dcterms]] so this information can be made
 					available to users.</li>
-				<li>If the standard does not define conformance values for the <code>conformsTo</code> property, EPUB
-					creators should use a URL [[url]] to where the standard is publicly available so users can look up
-					the specific details of the standard.</li>
+				<li>If the standard does not define conformance values for use in the <code>conformsTo</code> property,
+					the <code>conformsTo</code> property should include a URL [[url]] to where the standard is publicly
+					available so users can look up the specific details of the optimization.</li>
 				<li>If the identifier is not sufficient for a user to understand conformance (e.g., the guidelines are
-					not publicly available), EPUB creators should provide additional information about they have
-					optimized the content in the <a href="#confreq-schema-accessibilitySummary">accessibility
-						summary</a>.</li>
+					not publicly available), the metadata should provide additional information about the content has
```

Comment:
SHOULD caps? (not sure)

Reply:
Same as my previous comment about this being informative.